lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -539,9 +575,20 @@ class _PointerAdapter extends _BaseAdapter with _WheelEventListenerMixin {
         _convertEventsToPointerData(data: pointerData, event: event, details: move);
       }
       _callback(pointerData);
-    }, acceptOutsideGlasspane: true);
+    });
+
+    _addPointerEventListener(glassPaneElement, 'pointerleave', (html.PointerEvent event) {
```

Comment:
There's actually a comment on `addEventListener` that briefly explains why and gives general advice on what event types to register on which surface. Is that sufficient, or should I add more to it?

```dart
  /// Generally speaking, down and leave events should use [glassPaneElement]
  /// as the [target], while move and up events should use [html.window]
  /// instead, because the browser doesn't fire the latter two for DOM elements
  /// when the pointer is outside the window.
```

Should I just copy this explanation over, perhaps?